src/tests/ftest/server/metadata.yaml

```diff
@@ -15,41 +16,42 @@ server_config:
       nr_xs_helpers: 4
       first_core: 0
       pinned_numa_node: 0
-      fabric_iface: ib0
+      #fabric_iface: ib0
```

Comment:
Debugging? Why is this removed?

Reply:
oops, looks like I inadvertently pushed a version of this file that I use in a tmpfs only and non-IB development setup. Will fix this up in another push.

Reply:
Fixed. Now just the intended diffs to this file are found in the latest code on the branch.